### ISSUE TITLE
Show non-subscribed channels by default

### ISSUE THREAD

**Issue by dagwieers** [add-ons/plugin.video.tvvlaanderen#14]
There is a benefit of showing all the channels offered by TV Vlaanderen, even outside your subscription, just as the website and the app are doing. We could have a setting to disable this, and show the channels greyed out in the channel-overview, and when searching, when enabled. I would enable it by default too.

The biggest benefit is that TV Vlaanderen users know they can upgrade to have access to something they were looking for.

**Comment by michaelarnauts** [comment]
The app and site only do this when you are not logged in. I would not do this always as the list is really long (700+ channels), and I fetch the EPG for them in batches of 40 channels, so this would mean a lot of extra calls and quite some delay in opening the channel layout.

Also, most of these channels are not possible to see, even with an upgrade. (With the best subscription, you get 70-ish channels IIRC).

I do agree that the current behaviour to show an emptu listing when not logged in is also confusing, my suggestion is to ask the user to enter credentials when opening the channel listing when there are no credentials.

**Comment by dagwieers** [comment]
No, I am not talking about *all* channels, just the channels that would be available with an upgrade. Both the app and website show all channels from upgrade subscriptions, and if you try to play, indicate it requires another subscription plan. (I wanted to add a screenshot of the app but the app does not allow screenshots in some places).

See the Eleven channels in the screenshot I could take.
![Screenshot_20200830-144730](https://user-images.githubusercontent.com/388198/91659464-fd7a4500-eacf-11ea-9f56-944733b7f2de.jpg)

**Comment by dagwieers** [comment]
And if you search for content, you also get results from the channels you are not (yet) subscribed to. I think that gives a better experience as people would understand their options.
![Screenshot_20200830-145243](https://user-images.githubusercontent.com/388198/91659544-a1fc8700-ead0-11ea-8fe9-063abf20ff6a.jpg)
